web/src/main/java/se/inera/intyg/rehabstod/service/export/xlsx/XlsxExportServiceImpl.java

```diff
@@ -101,13 +100,15 @@
         addFilterMainHeader(sheet, rowNumber++, VALDA_FILTER);
 
         addFilterHeader(sheet, rowNumber++, FILTER_TITLE_FRITEXTFILTER, notEmpty(req) ? req.getFritext() : "-");
-        addFilterHeader(sheet, rowNumber++, FILTER_TITLE_VISAPATIENTUPPGIFTER, req.isShowPatientId() ? " Ja" : " Nej");
+        addFilterHeader(sheet, rowNumber++, FILTER_TITLE_VISAPATIENTUPPGIFTER, req.isShowPatientId() ? "Ja" : "Nej");
         addFilterHeader(sheet, rowNumber++, FILTER_TITLE_VALD_ALDER,
                 req.getAldersIntervall().getMin() + " - " + req.getAldersIntervall().getMax() + " år");
         rowNumber = addDiagnosKapitel(sheet, rowNumber++, FILTER_TITLE_VALDA_DIAGNOSER, req.getDiagnosGrupper()); // NOSONAR
         addFilterHeader(sheet, rowNumber++, FILTER_TITLE_VALD_SLUTDATUM, getFilterDate(req.getSlutdatumIntervall()));
         addFilterHeader(sheet, rowNumber++, FILTER_TITLE_VALD_SJUKSKRIVNINGSLANGD, getLangdintervall(req.getLangdIntervall()));
-        rowNumber = addLakareList(sheet, rowNumber++, FILTER_TITLE_VALDA_LAKARE, req.getLakare(), user); // NOSONAR
+        if (user.getUrval() != Urval.ISSUED_BY_ME) {
+            rowNumber = addLakareList(sheet, rowNumber++, FILTER_TITLE_VALDA_LAKARE, req.getLakare(), user); // NOSONAR
```

Comment:
Det här ser lite konstigt ut. Här görs en "post increment" av rowNumber efter att det skickats in som argument till addLakareList, fast direkt därefter sätts rowNumber till resultatet av anropet till addLakareList. Om jag tolkar det rätt så bör alltså inte "post increment" av variabeln ha någon betydelse och då är det nog bättre att ta bort den helt för att undvika förvirring.

Reply:
Nu är det ändrat.